### PR TITLE
Add conda package for point_cloud_utils

### PR DESCRIPTION
**Pull request by fwilliams**

### CODE REVIEW

**Comment by fwilliams** [comment]
@conda-forge/staged-recipes This seems ready to be made into a feedstock repo

File: recipes/example/meta.yaml [hunk 1]
```diff
@@ -3,19 +3,20 @@
 # Jinja variables help maintain the recipe as you'll update the version only here.
```
Comment (chrisburr) [inline recipes/example/meta.yaml]: Can you revert the changes to the example.

File: recipes/point_cloud_utils/meta.yaml [hunk 2]
```diff
@@ -0,0 +1,89 @@
+# Note: there are many handy hints in comments in this example -- remove them when you've finalized your recipe
```
Comment (chrisburr) [inline recipes/point_cloud_utils/meta.yaml]: Can you clear the various comments that are inherited from the example.

File: recipes/point_cloud_utils/meta.yaml [hunk 3]
```diff
@@ -0,0 +1,89 @@
...
+  # By default, the package will be built for the Python versions supported by conda-forge and for all major OSs.
+  # Add the line "skip: True  # [py<35]" (for example) to limit to Python 3.5 and newer, or "skip: True  # [not win]" to limit to Windows.
+  # Note: --no-deps is currently required due to https://github.com/conda/conda-build/issues/3254
+  # Once resolved, it should be removed.
+  script: "{{ PYTHON }} -m pip install . --no-deps -vv"
+  skip: true  # [win]
+  
+requirements:
+  build:
+    - {{ compiler('cxx') }}
```
Comment (chrisburr) [inline recipes/point_cloud_utils/meta.yaml]: The requirements don't need to be duplicated between `build` and `host`. The split should probably be something like:
```yaml
requirements:
  build:
    - {{ compiler('c') }}
    - {{ compiler('cxx') }}
    - cmake
    - git
  host:
    - python
    - pip
    - setuptools
    - numpy
    - scipy
```

File: recipes/point_cloud_utils/meta.yaml [hunk 4]
```diff
@@ -0,0 +1,89 @@
...
+    - pip
+    - setuptools
+    - git
+    - cmake
+    - numpy
+    - scipy
+    
+  run:
+    - python
+    - numpy
```
Comment (chrisburr) [inline recipes/point_cloud_utils/meta.yaml]: See [here](https://cf-docs-restructure.readthedocs.io/en/docs_restructure/maintainer/knowledge_base.html#linking-numpy).
```suggestion
    - {{ pin_compatible('numpy') }}
```

**Comment by fwilliams** [comment]
@chrisburr I made all the changes you requested and it looks like the CI is green. Let me know if there's anything else I need to do to merge. 

Thanks!

Francis
